packages/components/src/tabs/styles.ts

```diff
@@ -22,36 +22,57 @@ export const TabListWrapper = styled.div`
 		text-align: start;
 	}
 
-	@media not ( prefers-reduced-motion: reduce ) {
+	@media not ( prefers-reduced-motion ) {
 		&.is-animation-enabled::after {
-			transition-property: left, top, width, height;
+			transition-property: transform;
 			transition-duration: 0.2s;
 			transition-timing-function: ease-out;
 		}
 	}
+	--direction-factor: 1;
+	--indicator-start: var( --indicator-left );
+	&:dir( rtl ) {
+		--direction-factor: -1;
+		--indicator-start: var( --indicator-right );
+	}
 	&::after {
 		content: '';
 		position: absolute;
 		pointer-events: none;
+		transform-origin: calc( ( 1 - var( --direction-factor ) ) * 50% ) 0%;
```

Comment:
This may be tricky to understand for a developer with fresh eyes — should we add a comment explaining that, based on the writing direction, the x-axis transform origin may be either `0%` or `100%` ?

Reply:
Good point. Honestly I think I'm just gonna use another variable (something like `direction-x-origin`) and explicitly write the value. Readable > smart.

Reply:
I did it, let me know what you think.